### PR TITLE
fixed #267

### LINKED ISSUE #267: local_storage not working as expected
#234 added local storage capability.  This works partially

```  local_storage = {
      controller = [{
        device_slot =  "Embedded"
        drive_write_cache = "Unmanaged"
        initialize = true
        mode = "Mixed"
        predictive_spare_rebuild = "Unmanaged"
        logical_drives = [{
          accelerator = "Unmanaged"
          bootable = "true"
          # drive_technology = "SasHdd"
          name = "TestLd"
          num_physical_drives = 2
          raid_level = "RAID1"
        }]
      }]
    }
```

The above works when initialize is true but not when initialize = false.

when false:

1 error occurred:
        * module.spt_g10_trading_eg4_rsiegelTest.oneview_server_profile_template.spt: 1 error occurred:
        * oneview_server_profile_template.spt: Error in response: An unexpected error occurred.
 Response Status: 400 Bad Request
 Response Details: An unexpected and unhandled runtime exception occurred.

@AsisBagga @nabhajit-ray

### PR DESCRIPTION
**Pull request by AsisBagga**
### Description
#267 Fix for false value as Boolean attribute in local storage SPT and SP 
#268 Fixed Local Storage Issue for SP through SPT
#238 Fixed server profile connection settings issue.

### Issues Resolved
#267 
#268 
#238 

### Check List
- [ ] New functionality includes testing.
  - [ ]  All tests pass for go 1.11 + gofmt checks.
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: oneview/resource_server_profile_template.go [hunk 1]
```diff
@@ -799,8 +799,9 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 		serverProfileTemplate.Boot.ManageBoot = true
 		serverProfileTemplate.Boot.Order = bootOrder
 		rawBootMode := d.Get("boot_mode").(*schema.Set).List()[0].(map[string]interface{})
+		manage_mode := rawBootMode["manage_mode"].(bool)
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply oneview/resource_server_profile_template.go]: done

File: oneview/resource_server_profile_template.go [hunk 2]
```diff
@@ -823,8 +824,9 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 					Value: rawOverriddenSettingItem["value"].(string),
 				})
 			}
+			manage_bios := rawBiosItem["manage_bios"].(bool)
```
[the remark this answers is not in the document]
Comment (AsisBagga) [reply oneview/resource_server_profile_template.go]: done

**Comment by rismoney** [comment]
This looks to have fixed both issues.  Thank you very much for the quick turnaround!

**Comment by rismoney** [comment]
one final question - after merge, will this oneview_server_profile capability be available on the TF12 branch as well?  

I'm going to need to test oneview_server_profile inside a module, and in order to properly pass in network properties, I'll need v0.12 due to the data structures involved not being supported on 0.11.

**Comment by AsisBagga** [comment]
> one final question - after merge, will this oneview_server_profile capability be available on the TF12 branch as well?
> 
> I'm going to need to test oneview_server_profile inside a module, and in order to properly pass in network properties, I'll need v0.12 due to the data structures involved not being supported on 0.11.

This Pull Request will be merged to branch which supports Terraform v0.11.x . However, I will update this solution in another PR (another branch), which will be merged to [Terraform-12](https://github.com/HewlettPackard/terraform-provider-oneview/tree/Terraform-0.12) branch post review, by EOD.

**Comment by rismoney** [comment]
> > one final question - after merge, will this oneview_server_profile capability be available on the TF12 branch as well?
> > I'm going to need to test oneview_server_profile inside a module, and in order to properly pass in network properties, I'll need v0.12 due to the data structures involved not being supported on 0.11.
> 
> This Pull Request will be merged to branch which supports Terraform v0.11.x . However, I will update this solution in another PR (another branch), which will be merged to [Terraform-12](https://github.com/HewlettPackard/terraform-provider-oneview/tree/Terraform-0.12) branch post review, by EOD.

Are these profile related changes in TF12?  Trying to just clarify what was merged to master because i saw 3000+ files changed.

**Comment by AsisBagga** [comment]
> > > one final question - after merge, will this oneview_server_profile capability be available on the TF12 branch as well?
> > > I'm going to need to test oneview_server_profile inside a module, and in order to properly pass in network properties, I'll need v0.12 due to the data structures involved not being supported on 0.11.
> > 
> > 
> > This Pull Request will be merged to branch which supports Terraform v0.11.x . However, I will update this solution in another PR (another branch), which will be merged to [Terraform-12](https://github.com/HewlettPackard/terraform-provider-oneview/tree/Terraform-0.12) branch post review, by EOD.
> 
> Are these profile related changes in TF12? Trying to just clarify what was merged to master because i saw 3000+ files changed.

master was recently upgrade to TF version 0.12.
This PR will be added to Terraform-0.11 Branch

**Review by nabhajit-ray — APPROVED** [review]

**Review by SHANDCRUZ — APPROVED** [review]

**Pull request merged**